diff --git a/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.h b/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.h
index 9cd24c6..f6d17a6 100644
--- a/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.h
+++ b/node_modules/expo-modules-core/ios/Fabric/ExpoFabricViewObjC.h
@@ -2,7 +2,9 @@
 
 #ifdef RCT_NEW_ARCH_ENABLED
 
+#if __has_include(<UIKit/UIKit.h>)
 #import <UIKit/UIKit.h>
+#endif
 #import <React/React-Core-umbrella.h>
 
 #ifdef __cplusplus
