diff --git a/node_modules/react-native-macos/React/Views/RCTView.m b/node_modules/react-native-macos/React/Views/RCTView.m
index 294a940..d72f582 100644
--- a/node_modules/react-native-macos/React/Views/RCTView.m
+++ b/node_modules/react-native-macos/React/Views/RCTView.m
@@ -764,7 +764,11 @@ RCT_NOT_IMPLEMENTED(-(instancetype)initWithCoder : unused)
 -(void)didUpdateShadow
 {
     NSShadow *shadow = [NSShadow new];
-    NSColor *baseShadowColor = [NSColor colorWithCGColor:_shadowColor];
+    
+    NSColor *baseShadowColor = nil;
+    if (_shadowColor != NULL) {
+      baseShadowColor = [NSColor colorWithCGColor:_shadowColor];
+    }
     shadow.shadowColor = [baseShadowColor colorWithAlphaComponent:[self shadowOpacity]];
     shadow.shadowOffset = [self shadowOffset];
     shadow.shadowBlurRadius = [self shadowRadius];
